charts/docker-mailserver/templates/certificate.yaml

```diff
@@ -1,6 +1,6 @@
-{{- if not .Values.ssl.useExisting -}}  
+{{- if not .Values.ssl.useExisting -}}
 ---
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: certmanager.k8s.io/v1alpha1
```

Comment:
I ran into a problem with this before. Did we upgrade cert-manager?

Reply:
We did not upgrade it yet. There is a story in Stormwatch's backlog to do it. We are on .7, and I believe .12 is out.